### ISSUE TITLE
No easy way to get sharing link from UI alone

### ISSUE THREAD

**Issue by mitko** [excalidraw/excalidraw#5376]
In a normal browser I can just copy the sharing link from the address bar.

But I'm using a chrome app, because I use Excalidraw so much. I'm using the paid version of excalidraw with my teammate.
I want to be able to share a certain drawing with them, but there is no simple way to do so. 
From the share menu of the drawign:

<img width="191" alt="image" src="https://user-images.githubusercontent.com/125253/176818598-2930e7ee-4af0-4e85-91e0-9b6b9b39ff5a.png">

there is no option to get a sharing link for members of the workspace only

<img width="465" alt="image" src="https://user-images.githubusercontent.com/125253/176818654-45c58385-2090-4593-8d82-969b1b964258.png">

Can you please add copy link option which doesn't make the page public? Here's how Google does it for example:

<img width="499" alt="image" src="https://user-images.githubusercontent.com/125253/176818838-0330dc2b-6a0b-4ba2-95cd-6b2d05a096a8.png">

**Comment by dwelle** [comment]
@mitko good point about the missing url bar in installed apps, and in general it's a good idea to make the UI similar to what you suggest. We'll take a look!

**Comment by mitko** [comment]
Thanks @dwelle ! I love excalidraw

**Comment by Satindar31** [comment]
Tell now even there is no easy way to share the link.
![image](https://github.com/excalidraw/excalidraw/assets/73517520/1cbd1238-ce6c-4771-a054-2f7275aff174)
